src/Features/LanguageServer/Protocol/Handler/References/FindUsagesLSPContext.cs

```diff
@@ -195,13 +195,16 @@ public override async ValueTask OnReferenceFoundAsync(SourceReferenceItem refere
                 return null;
             }
 
+            var conversionService = _workspace.Services.GetRequiredService<IGlyphConversionService>();
+            var guidAndId = conversionService.ConvertToImageId(definitionGlyph);
+
             // TO-DO: The Origin property should be added once Rich-Nav is completed.
             // https://github.com/dotnet/roslyn/issues/42847
             var result = new VSInternalReferenceItem
```

Comment:
Note (for a followup) - the returned item will be entirely different for the non-VS one.   The spec only supports simple locations
https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_references

Reply:
thanks :)